src/main/java/de/tum/in/www1/artemis/repository/metis/AnswerPostRepository.java

```diff
@@ -0,0 +1,23 @@
+package de.tum.in.www1.artemis.repository.metis;
+
+import javax.validation.constraints.NotNull;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import de.tum.in.www1.artemis.domain.metis.AnswerPost;
+import de.tum.in.www1.artemis.web.rest.errors.EntityNotFoundException;
+
+/**
+ * Spring Data repository for the AnswerPost entity.
+ */
+@SuppressWarnings("unused")
```

Comment:
I did not add/change this. It was used in the former `StudentQuestionAnswerRepository` before.

Reply:
If I understand that correctly, this is used to avoid compiler warnings for unused local variables. As it is used in many other repositories, I think it's reasonable?

Reply:
Ok.